.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+Documentation:
+  Enabled: false
+LineLength:
+  Max: 99
```

Comment:
You can add:
```
Style/FrozenStringLiteralComment:
  Enabled: false
```
To avoid `frozen_string_literal: true`

Reply:
Nice 👍